plugins/modules/vmware_host_vmnic_info.py

```diff
@@ -18,6 +18,7 @@
 - If C(cluster_name) is provided, then vmnic information about all hosts from given cluster will be returned.
 - If C(esxi_hostname) is provided, then vmnic information about given host system will be returned.
 - Additional details about vswitch and dvswitch with respective vmnic is also provided which is added in 2.7 version.
+- Additional details about lldp added in 1.10.0
```

Comment:
```suggestion
- Additional details about lldp added in 1.11.0
```

I confirmed that the module works well on the vSphere environment with LLDP.  
Version 1.10.0 has alredy released, so could you please change 1.10.0 to 1.11.0?  
I'll merge this after confirming it is changed.

Reply:
@sky-joker I've updated the version.